Package.swift

```diff
@@ -30,15 +30,16 @@ let package = Package(
     ],
     dependencies: [
         .package(name: "SmokeAWSGenerate",
-                 url: "https://github.com/amzn/smoke-aws-generate.git", .branch("main")),
+                 url: "https://github.com/amzn/smoke-aws-generate.git", .branch("working")),
```

Comment:
nit: make sure we switch these branches back to `main` before merging.

Reply:
Will do.